### ISSUE TITLE
Bundles directory directory doesn't exist anymore 

### ISSUE THREAD

**Issue by rmpestano** [rtfpessoa/diff2html#310]
In [distributions](https://github.com/rtfpessoa/diff2html#distributions) section all the links for `Manually download and import` gives **404** because `bundles` directory doesn't exist anymore.

**Comment by azimut2000** [comment]
+1

I have also problem to understand how your LIB can be use....

In the documentation, you said "Check out for the demo" that is a link on https://github.com/rtfpessoa/diff2html/blob/master/docs/demo.html but this page do not exist !
I also have problem to use npm because i have proxy in my compagny and i have had a lot of difficulties to find file like "diff2html.min.js" or "diff2html.min.css" on Internet : is it possible to make a release of diff2html with theses files ? 

Thank's to all contibutors,
Thierry

**Comment by rmpestano** [comment]
Hi @azimut2000 , for past versions like 2.x you can find the dist folder on the tag, [see here](https://github.com/rtfpessoa/diff2html/tree/2.12.2/dist).

I hope this issue gets fixed so we can grab the latest versions without needing to build locally

**Comment by azimut2000** [comment]
Thank's for link...
Have you an example / demo page for using thees 2.x .js ?
Have you a page for the 3.x version too ? What are differents between theses 2 versions ? 
Thank's again
Thierry

**Comment by rtfpessoa** [comment]
Sorry for the delay getting here. I have been recovering from a cirurgy and mostly away from the computer.

I did some quick tests and if I install the latest version as I do in `diff2html-cli` everything seems to be there including the bundles.

@rmpestano @azimut2000 this is a case where I need to improve the documentation.

TLDR, I removed all build outputs from the repository and now they are only published to npm, since its weird to version them here.

I left the links for docs but they only exist if you build, which is probably weird now that I think about it.

You can start using something like [jsdelivr](https://www.jsdelivr.com/) to access the files. For example: `https://cdn.jsdelivr.net/npm/diff2html@3.1.2/bundles/css/diff2html.min.css`

Hope this helps, I will try to update the docs soon to remove the confusion.

**Comment by rtfpessoa** [comment]
@azimut2000 about the demo you can check `https://diff2html.xyz/demo.html` for a live example.

**Comment by rtfpessoa** [comment]
@azimut2000 @rmpestano if you have any more suggestions for improvements and how to make the docs more approachable. Please let me know or send a PR.

**Comment by azimut2000** [comment]
Thank's a lot Radrigo...
Take time for you, it's the most important !
Ok for suggestions... I will send you as soon as i will finish my project.

**Comment by rmpestano** [comment]
Hi Rodrigo,

I think just direct links to the binaries is enough on the documentation so the ones not using npm can benefit as well.

I hope everything is ok with you, take care!
